tests/Fixtures/TestBundle/Model/Chair.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Tests\Fixtures\TestBundle\Model;
+
+use ApiPlatform\Metadata\Resource\Factory\NotExposedOperationResourceMetadataCollectionFactory;
+
+/**
+ * Single resource without identifiers and with a single collection.
+ * A NotExposed operation with "routeName: api_genid" is automatically added to this resource.
+ *
+ * @see NotExposedOperationResourceMetadataCollectionFactory::create()
+ *
+ * @author Vincent Chalamon <vincentchalamon@gmail.com>
+ */
+class Chair
```

Comment:
Why not using annotations? It's easier to maintain than YAML.

Reply:
Because this directory seems already loaded by the test configuration and we can't make a difference with v3 configuration vs legacy configuration. Instead, we load different directories depending on the `metadata_backward_compatibility_layer`.